nexus/catalog/migrations/V27__drop_stats_trigger.sql

```diff
@@ -0,0 +1,26 @@
+CREATE OR REPLACE FUNCTION clear_mirror_stats() RETURNS TRIGGER AS $$
+DECLARE
+    mirrorName TEXT;
+    cloneMirrorRegex TEXT;
+BEGIN
+    mirrorName := NEW.name;
+    cloneMirrorRegex := 'clone_' || mirrorName || '_%';
```

Comment:
```suggestion
    cloneMirrorPattern := 'clone\\_' || replace(replace(mirrorName, '_', '\\_'), '%', '\\%') || '\\_%';
```
_ is a single character wildcard in like patterns. Need to escape. https://www.postgresql.org/docs/12/functions-matching.html